content/blogposts/2023/guest-post-arafat-khan-php-python.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Climbing Mount Everest in Flip-flops: My PHP journey as a Python dev"
+authors:
+  - name: Arafat Khan
+    url: https://twitter.com/arafatkatze
+  
+publishDate: 2023-11-21T10:00-01:00
+description: "When tasked with creating a printing API in PHP despite having zero experience, Arafat Khan leveraged an AI coding assistant named Cody to transform what felt like an Everest climb into a swift 3 day project that resulted in a fully functioning API."
```

Comment:
When tasked with creating a printing API in PHP despite having zero experience, Arafat Khan leveraged Cody to transform what felt like an Everest climb into a swift 3 day project that resulted in a fully functioning API.